pkgs/applications/networking/cluster/atlantis/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
looks like the repo has a go.mod, did you want try making this into a buildGoModule? Should remove need for deps.nix

Reply:
That is my very first packaging of a go package. Actually my very first interaction with go at all, if one can consider this an interaction. I wasn't even aware this was possible! I'll change that immediately, better not commit thousands of useless lines of text.

Reply:
https://nixos.org/nixpkgs/manual/#ssec-go-modules

Reply:
Fixed it, and actually read the nixpkgs manual section about it, I'm too used to not rely on documentation, so I missed this. Should be better now!

Reply:
that's the point of reviews :) don't worry about it

Reply:
and it's only applicable to projects that converted over modules